README.md

```diff
@@ -1,76 +1,32 @@
-	* more opinionated than a bespoke architecture, in exchange for greater knowledge/code sharing between teams
-	* more isolated than GOV.UK PaaS, but encourages sharing as much as possible 
-
-
-## Getting started
-
-Come talk to the Automate team. We're on Slack in #re-autom8 and on the east side of the 6th floor.
-
-You also might want to look at how other teams are using GSP:
-
-- [Verify eIDAS proxy node](https://github.com/alphagov/verify-proxy-node/tree/master/ci)
-- [Verify Document Checking Service (private repo)](https://github.com/alphagov/doc-checking)
-
-## Contributing
-
-Contributions encouraged!
-
-Changes require commits signed by [GDS Trusted Developers](https://github.com/alphagov/gds-trusted-developers)
-
-## Help and support
-
-The platform is maintained by [GDS Reliability Engineering](https://reliability-engineering.cloudapps.digital/) and support for GDS service teams is provided according to the [Technology & Operations Shared Responsibility Model](https://reliability-engineering.cloudapps.digital/documentation/strategy-and-principles/techops-shared-responsibility-model.html)
-
-For help or support:
-
-- read our [documentation](/docs)
-- raise an [issue](https://github.com/alphagov/gsp/issues)
-- message the team on the Reliability [Engineering Slack channel](https://gds.slack.com/messages/CAD6NP598) [#reliability-eng](https://gds.slack.com/messages/CAD6NP598)
-
-
+GSP ([GDS](https://www.gov.uk/government/organisations/government-digital-service) Supported Platform) was a Kubernetes distribution based on Amazon EKS.
+
+Technically:
+* The Kubernetes/EKS version is behind - we're on 1.16 and the latest is 1.20. 1.16 will not be possible to use after July 2021.
+ * There's a TODO in `pipelines/deployer/deployer.yaml` about k8s 1.15 we can probably remove.
+* GSP relies on Istio 1.5.8 which became end-of-life on 2020-08-24. Also 1.6 was end-of-life on 2020-11-23, 1.7 was end-of-life on 2021-02-25, and 1.8 will be end-of-life on 2021-05-12.
+* GSP ran Prometheus and Grafana through prometheus-operator 8.15.6, and it's no longer developed at `git@github.com:helm/charts.git` stable/prometheus-operator - the latest version of chart we were using is now deprecated.
+* The check-vulnerabilities job in each cluster deployment pipeline would find all sorts of things in the third-party images we used like cluster-autoscaler, concourse-web, external-dns, fluentd-cloudwatch, fluentd-kubernetes-daemonset, and more. Some of these may be resolvable by upgrading the version of the software used.
+* It's based on Terraform but with some weird extra CloudFormation that should be merged into the Terraform - `modules/k8s-cluster/data/nodegroup-v2.yaml` and (especially obscure, but possibly unnecessary depending on the item below) `modules/k8s-cluster/data/nodegroup.yaml`.
+* We're not completely certain that the cluster-management nodes still necessary, it may be possible to put gatekeeper and the cluster-autoscaler on normal worker nodes.
```

Comment:
```suggestion
* We're not completely certain that the cluster-management nodes are still necessary, it may be possible to put gatekeeper and the cluster-autoscaler on normal worker nodes.
```